program/c/src/oracle/upd_aggregate.h

```diff
@@ -102,19 +102,12 @@ static void upd_ema(
     // compute numer/denom and new value from decay factor
     pd_load( numer, ptr->numer_ );
     pd_load( denom, ptr->denom_ );
-    if ( numer->v_ < 0 || denom->v_ < 0 ) {
```

Comment:
this code was a temporary thing when we changed up the ema early on. doesn't need to exist now, and also prevents negative ema values from working properly.